File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.c
#include <string.h>

int finalValueAfterOperations(char ** operations, int operationsSize){
    int val = 0;
    for(int i = 0; i < operationsSize; i++)
    {
        if (strcmp(operations[i], "++X") == 0 || strcmp(operations[i], "X++") == 0)
            val++;
        else val--;   
    }
    return val;
}

File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.cs
public class Solution {
    public int FinalValueAfterOperations(string[] operations) {
        int val = 0;
        foreach (string op in operations)
        {
            if (op.Equals("++X") || op.Equals("X++")) val++;
            else val--;
        }
        return val;
    }
}